certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -3720,55 +4440,56 @@
               "properties": {
                 "docContent": {
                   "type": "string",
-                  "description": "PDF file content.",
+                  "description": "Map source File Content from the source action",
                   "title": "PDF File content",
                   "format": "byte",
                   "x-ms-visibility": "important"
                 },
                 "docName": {
                   "type": "string",
-                  "description": "PDF File Name",
+                  "description": "Source file name with file extension",
                   "title": "PDF File Name",
                   "x-ms-visibility": "important"
                 },
                 "pageNumberFormat": {
                   "type": "string",
                   "description": "Page number format to be displayed",
-                  "title": "Page Format"
+                  "title": "Page Format",
+                  "default": "# of {totalPages}"
                 },
                 "alignX": {
-                  "default": "left",
-                  "description": "Set alignment in horizontal scale",
+                  "default": "Left",
+                  "description": "Select the Horizontal Alignment",
                   "enum": [
-                    "left",
-                    "center",
-                    "right"
+                    "Left",
+                    "Center",
+                    "Right"
                   ],
                   "title": "Horizontal Alignment",
                   "type": "string",
                   "x-ms-visibility": "important"
                 },
                 "alignY": {
-                  "default": "top",
-                  "description": "Set alignment in vertical scale",
+                  "default": "Top",
+                  "description": "Select the Vertical Alignment",
                   "enum": [
-                    "top",
```

Comment:
@varunynoox An enum values has been changed from the input field. Existing flows may have these value chosen and will continue to send the old values.
Can you please confirm that no users will be affected by this change?

Reply:
sending old values of enum will not affect any user